examples/aiohttp-echo/app.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may
+#  not use this file except in compliance with the License. You may obtain
+#  a copy of the License at
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+
+from __future__ import unicode_literals
```

Comment:
Is this necessary?

`async` and `await` keywords are only valid for Python 3.5 or over. So, I guess that this module doesn't need to care about the compatibility between 2.7 and 3.x, even if `line-bot-sdk-python` supports 2.7. It should be failed with 2.7.

Reply:
I couldn't understand what you were saying.

Reply:
Ah, I got it. You said "`from __future__ import unicode_literals` is completely useless on Python 3+", right?

Reply:
Yes. `from __future__ import unicode_literals` is for migration use both Python 2/3. But, this module works on Python 3.5+, not 2.7.

Reply:
> Is this necessary?
> 
> 
> 
> `async` and `await` keywords are only valid for Python 3.5 or over. So, I guess that this module doesn't need to care about the compatibility between 2.7 and 3.x, even if `line-bot-sdk-python` supports 2.7. It should be failed with 2.7.